src/Site_Command.php

```diff
@@ -498,6 +769,20 @@ private function create_site( $assoc_args ) {
 	 */
 	private function delete_site() {
 
+		// Commented below block intentionally as they need change in DB 
```

Comment:
Do not add any commented code. If you need it you can always take it back from commit or move it to another temp branch if you need it as ref.